numpy/_core/shape_base.py

```diff
@@ -386,8 +390,10 @@ def stack(arrays, axis=0, out=None, *, dtype=None, casting="same_kind"):
 
     Parameters
     ----------
-    arrays : sequence of array_like
-        Each array must have the same shape.
+    arrays : ndarray or sequence of ndarrays
+        Each array must have the same shape. In the case of a single ndarray
+        input, it will be treated as a sequence of `shape[0]` arrays, each of
+        shape `shape[1:]`.
```

Comment:
```suggestion
        array_like input, it will be treated as a sequence of arrays; i.e.,
        each element along the zeroth axis is treated as a separate array.
```